src/uwtools/api/driver.py

```diff
@@ -93,46 +95,50 @@ def execute(
     return True
 
 
-def tasks(module: str, classname: str) -> dict[str, str]:
+def tasks(module: Union[Path, str], classname: str) -> dict[str, str]:
     """
     Returns a mapping from task names to their one-line descriptions.
 
     :param module: Name of driver module.
     :param classname: Name of driver class to instantiate.
     """
-    if not (class_ := _get_driver_class(module, classname)):
+    class_, _ = _get_driver_class(module, classname)
+    if not class_:
         log.error("Could not get tasks from class %s in module %s", classname, module)
         return {}
     return _tasks(class_)
 
 
-def _get_driver_class(module: Union[Path, str], classname: str) -> Optional[Type]:
+def _get_driver_class(
+    module: Union[Path, str], classname: str
+) -> tuple[Optional[Type], Optional[Path]]:
     """
     Returns the driver class.
 
     :param module: Name of driver module to load.
     :param classname: Name of driver class to instantiate.
     """
-    module = str(module)
-    if not (m := _get_driver_module_explicit(module)):
-        if not (m := _get_driver_module_implicit(module)):
+    if not (m := _get_driver_module_explicit(Path(module))):
+        if not (m := _get_driver_module_implicit(str(module))):
             log.error("Could not load module %s", module)
-            return None
+            return None, None
+    assert m.__file__ is not None
+    module_path = Path(m.__file__)
```

Comment:
Here's where we can now discover the path to the module file and pass it back to the caller so that it can be used to find the sibling `.jsonschema` file.